theano/tensor/opt.py

```diff
@@ -1487,12 +1488,20 @@ def local_cast_cast(node):
 class Assert(T.Op):
     """
     Implements assertion in a computational graph.
+    Returns the first parameter if the condition is true, otherwise, trigger
+    AssertionError.
+
+    Example:
+      T = theano.tensor
+      x = T.vector('x')
+      assert_op = T.opt.Assert()
+      func = theano.function([x], assert_op(x, x.size<2))
 
     Notes:
-    This Op can be removed from the graph because of optimizations, and can
-    hide some possible optimizations to the optimizer.
-    Also, the output of the Op must be returned by the function computing the
-    graph, otherwise it will not be used.
+    This Op is an debugging feature. It can be removed from the graph
+    because of optimizations, and can hide some possible optimizations to the
```

Comment:
just want to make sure,  it's also true that removing is result from optimization?